src/jobs/README.md

```diff
@@ -191,16 +193,19 @@ approvedUsers: {
-ATTESTATION_UNIT_OF_TIME=years
-DAYS_TO_ATTESTATION=45
-```
+The [src/config.ts](../config.ts) file contains default values so the application will run if these values are not provided in the environment. However, all date calculations are done via [moment.js](https://momentjs.com/), so there is a validation for any configured "unit of time" (`ATTESTATION_UNIT_OF_TIME`, `EXPIRY_UNIT_OF_TIME`) values to ensure they align with Moment's expected [Duration types](https://github.com/moment/moment/blob/develop/ts3.1-typings/moment.d.ts#L314). If an unexpected value is provided, the application will fail to start.\*
+
+> **\*Note**: For our purposes, this validation excludes values smaller than `"days"` as shorter timespans are not feasible.
+
+> **Important Note**: The `expiresAtUtc` value is saved in the DB application document, so it is important to be aware that changing the `EXPIRY_UNIT_OF_TIME` and/or `EXPIRY_UNIT_COUNT` after there are approved applications in the system would result in differing approval periods for applications. **This also means the value should not be changed in production.** The other values listed here are used to calculate fields that are not stored in the DB; however it is also not recommended to change them in a production environment, if there are existing approved applications.
+
+| Variable Name              | Required | Type                                  |   Default | Description                                                                                                                                                                                                                                                                                                                     |
+| -------------------------- | :------: | ------------------------------------- | --------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `ATTESTATION_UNIT_COUNT`   | Optional | number                                |       `1` | The number of units of time used to calculate an approved application's `attestationByUtc` value. Used in conjunction with `ATTESTATION_UNIT_COUNT`. An unattested application would be transitioned to `PAUSED` after this date.                                                                                               |
+| `ATTESTATION_UNIT_OF_TIME` | Optional | moment.unitOfTime.DurationConstructor | `"years"` | The unit of time used to calculate an approved application's `attestationByUtc` value. Used in conjunction with `ATTESTATION_UNIT_COUNT`. An unattested application would be transitioned to `PAUSED` after this date.                                                                                                          |
```

Comment:
can you link the `moment.unitOfTime.DurationConstructor` to somewhere where we can see the options? or just list the options?

Reply:
i just looked at this as github will render it... the long value makes the column widths weird... i would suggest just pasting in the list of permitted values, or have just `DurationConstructor` with a link to the valid values.

Reply:
I added a link to the moment types in the description above the table too